src/rules/pr.ts

```diff
@@ -31,4 +31,20 @@ export let pr = {
     }
   },
 
+  /** Warns if http:// was used instead of https:// */
```

Comment:
> I've put the code to detect it on pr.test, but actually I don't we have a convention to organize files/folders.

We kind of have this already for existing rules.

This, for example, is listed under "platform agnostic" on our README, so you could have added it [here](https://github.com/indigotech/dangerjs-plugin/blob/feature/http-test/src/rules/platform-agnostic.ts) instead of inside `pr`